src/burn/drv/pre90s/d_armedf.cpp

```diff
@@ -822,7 +821,7 @@ static INT32 DrvSynchroniseStream(INT32 nSoundRate)
 
 static INT32 DrvSyncDAC()
 {
-	return (INT32)(float)(nBurnSoundLen * (ZetTotalCycles() / (6000000.000 / (nBurnFPS / 100.000))));
+	return (INT32)(float)(nBurnSoundLen * (ZetTotalCycles() / (((6000.0 / nBurnFPS) * 6000000.000) / (nBurnFPS / 100.000))));
 }

```

Comment:
Just pointing out: either way returns exactly the same calculation :)

Reply:
Thanks! This code should have been revert.
Armed F refresh rate is 57 before the change, so pitch of DAC sound was wrong. This code was to fix it.
But later I noticed that the refresh rate was wrong and fixed it to 59.08 (same as MAME).
It seems that the DAC pitch was corrected only by changing refresh rate, and this code was needless.